crates/router/src/core/webhooks/webhook_events.rs

```diff
@@ -72,6 +78,32 @@ pub async fn list_initial_delivery_attempts(
                 _ => None,
             };
 
+            let now = common_utils::date_time::now();
+            let past_90_days_ago = now - time::Duration::days(EVENT_DELIVERY_LIMIT_DAYS);
+
+            let created_after = match created_after {
+                Some(created_after) => {
+                    if created_after < past_90_days_ago {
+                        Err(errors::ApiErrorResponse::InvalidRequestData { message: format!("You can only request this data from the past {EVENT_DELIVERY_LIMIT_DAYS} days or earlier.") })
```

Comment:
The "or earlier" would convey the wrong meaning, right? Do we need it? Moreover, we aren't clear about which field is not within range.

Let's try to mention something like:
```text
`created_after` must be a timestamp within the past 90 days
```